layouts/partials/footer.html

```diff
@@ -10,7 +10,7 @@
   <div class="container is-max-widescreen">
     <div id="footer-columns" class="columns">
       <div class="footer-logo-column">
-        <img id="footer-logo" src="{{ printf "/images/%s" $logo | relURL }}" alt="{{ $title }} logo. {{ $navbarLogo.altText }}">
+        <img id="footer-logo" src="{{ printf "/images/%s" $logo | relURL }}" alt="{{ $title }} logo. {{ $navbarLogo.altText }}" width="67" height="75">
```

Comment:
Is the intent to still have explicit dimensions here, or is the CSS above supposed to take care of it?

Reply:
Yes, the explicit dimensions in the IMG tag are the recommended best practice.  It informs the browser of the image's aspect ratio early in the rendering process, which helps prevent re-rendering as resources are loaded.  Later, the CSS sets the image's actual display size.